flang/include/flang/Parser/parse-tree.h

```diff
@@ -3738,6 +3738,29 @@ struct OmpOrderClause {
   std::tuple<std::optional<OmpOrderModifier>, Type> t;
 };
 
+// InteropType
```

Comment:
Add the standard ref numbers:
Eg: // Ref: [4.5:169-170], [5.0:255-256], [5.1:288-289], [5.2:323-324] 
for depend clause.

Reply:
Sure, will add it.

Reply:
Addressed the review comment on [#8d8dc3a](https://github.com/swatheesh-mcw/llvm-project/pull/1/commits/8d8dc3ac1dbfb6bc92802da7bd6504513ad60692)